experimental/SymmetricIntersections/Types.jl

```diff
@@ -0,0 +1,198 @@
+import Oscar: elem_type
+
+"""
+We work always with finite groups and splitting fields of characteristic zero.
+So by default, we take `QQAb` which is algebraically close of characteristic
+zero.
+"""
+
+export CharacterGrassmannian,
+       DeterminantGrassmannian,
+       ElevCtx,
+       InvariantGrassmannian,
+       IsotypicalGrassmannian,
+       LinRep,
+       ParameterMap,
+       ProjRep,
+       RepRing,
+       SymmetricGrassmannian,
+       SymmetricIntersections
+
+###############################################################################
+#
+#  Elevations context
+#
+###############################################################################
+
+@attributes mutable struct ElevCtx{T, U}
+  L::Vector{T}
+  d::Int
+  it::SubObjectIterator{PointVector{fmpz}}
+  f::U
+  bounds::Tuple{Vector{Int}, Vector{Int}}
+
+  function ElevCtx(L::Vector{T}, d::Int, it::SubObjectIterator{PointVector{fmpz}}, f::U, bounds::Tuple{Vector{Int}, Vector{Int}}) where {T, U}
+    z = new{T, U}()
+    z.L = L
+    z.d = d
+    z.it = it
+    z.f = f
+    z.bounds = bounds
+    return z
+  end
+end
+
+###############################################################################
+#
+#  Representations
+#
+###############################################################################
+
+### Representation ring
+
+"""
+Here a representation ring is a "parent" for linear representation, used to store
```

Comment:
This description seems to be missing its start?

Maybe like this?
```suggestion
A structure encoding a representation ring.

Here a representation ring is a "parent" for linear representation, used to store
```